hydra-node/test/Hydra/Chain/Direct/WalletSpec.hs

```diff
@@ -232,6 +236,30 @@ prop_removeUsedInputs =
           & counterexample ("Remaining UTXO: " <> show utxo')
           & counterexample ("Tx UTxO: " <> show txUTxO)
           & counterexample ("Wallet UTXO: " <> show walletUTxO)
+        
+--        
+-- estimateScriptsCost
+--
+
+prop_correctTxIsNotErroringOut :: Property
+prop_correctTxIsNotErroringOut = 
+  forAllBlind (reasonablySized genValidatedTx) $ \tx -> 
+    forAllBlind (reasonablySized $ genOutputsForInputs tx) $ \txUTxO ->
+      let scriptCost = estimateScriptsCost ledgerPParams systemStart epochInfo txUTxO tx 
+      in property (isRight scriptCost)
+    
+prop_failsWithScriptExecutionFailed :: Property
+prop_failsWithScriptExecutionFailed = 
+  forAllBlind (reasonablySized genValidatedTx) $ \tx -> 
+    forAllBlind (reasonablySized $ genOutputsForInputs tx) $ \txUTxO ->
+      let
+          -- TODO: update the tx here so that we can see the test failing
+          -- generated tx does not contain any script data
```

Comment:
It was approved.. now it requires changes again to address this TODO.

(I don't think these tests are worth their weight. Especially as I see a more holistic rewrite of the Wallet upcoming)

Reply:
Ok, I'll remove them from this pr then. Do you agree @ffakenz ?

Reply:
I would not remove the test, especially knowing we are going to refactor Wallet (i think they will come in handy).

But I don't have a strong opinion. Would leave for the rest to decide on this 🙏

Reply:
The test that was supposed to fail `prop_failsWithScriptExecutionFailed` was actually not implemented. And `prop_correctTxIsNotErroringOut` is green because the tx gen we are testing with is not actually using any scripts so `estimateScriptsCost` always produces `Right`. I'll nuke it for now.